<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-image: url(&quot;/home/jimz/Downloads/Ingenieria_Software/login.jpg&quot;);
}
QLineEdit{
	background: #ffffff;
}
QPushButton{
background: #bbcbe5;
}
</string>
  </property>
  <widget class="QLineEdit" name="passLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="loginComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Veterinaria</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Adm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Refugio</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="userLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>32</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passLineEdit</tabstop>
  <tabstop>loginComboBox</tabstop>
  <tabstop>loginPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
